### ISSUE TITLE
[Bug] REI Crashes after opening the inventory menu.

### ISSUE THREAD

**Issue by tomycabre** [shedaniel/RoughlyEnoughItems#922]
### What happened?

This is a problem with architectury (I THINK, NOT SURE) but as it is required to run this mod I use this repository and not the architectury one to solve this. 

### What mod loaders are you seeing the problem on?

Forge

### What do you think this bug is of?

- [ ] Visual
- [ ] Recipe Lookup
- [ ] Cheat Mode
- [X] Plugin Integration / JEI Plugin Compatibility
- [ ] Others

### Relevant log output

```shell
---- Minecraft Crash Report ----
// My bad.

Time: 6/15/22, 3:49 PM
Description: Adding entry

java.lang.NoSuchMethodError: 'net.minecraftforge.fluids.FluidAttributes net.minecraft.world.level.material.Fluid.getAttributes()'
	at dev.architectury.hooks.fluid.forge.FluidStackHooksImpl.getStillTexture(FluidStackHooksImpl.java:81) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at dev.architectury.hooks.fluid.FluidStackHooks.getStillTexture(FluidStackHooks.java) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.plugin.client.entry.FluidEntryDefinition$FluidEntryRenderer.getExtraData(FluidEntryDefinition.java:221) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at me.shedaniel.rei.plugin.client.entry.FluidEntryDefinition$FluidEntryRenderer.getExtraData(FluidEntryDefinition.java:210) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at me.shedaniel.rei.impl.client.gui.widget.BatchedEntryRendererManager.add(BatchedEntryRendererManager.java:77) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.widget.BatchedEntryRendererManager.addAll(BatchedEntryRendererManager.java:61) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.widget.BatchedEntryRendererManager.<init>(BatchedEntryRendererManager.java:55) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.widget.entrylist.EntryListWidget.m_6305_(EntryListWidget.java:314) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.ScreenOverlayImpl.renderWidgets(ScreenOverlayImpl.java:741) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.ScreenOverlayImpl.m_6305_(ScreenOverlayImpl.java:640) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.RoughlyEnoughItemsCoreClient.lambda$registerEvents$20(RoughlyEnoughItemsCoreClient.java:393) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at dev.architectury.event.forge.EventHandlerImplClient.eventDrawScreenEvent(EventHandlerImplClient.java:158) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at dev.architectury.event.forge.__EventHandlerImplClient_eventDrawScreenEvent_Post.invoke(.dynamic) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:93) ~[eventbus-6.0.0.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-6.0.0.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-6.0.0.jar%232!/:?] {}
	at net.minecraftforge.client.ForgeHooksClient.drawScreenInternal(ForgeHooksClient.java:383) ~[forge-1.19-41.0.33-universal.jar%2399!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:375) ~[forge-1.19-41.0.33-universal.jar%2399!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:896) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:classloading,pl:accesstransformer:B,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1071) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:661) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19-41.0.33.jar%2316!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:155) [bootstraplauncher-1.1.0.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at dev.architectury.hooks.fluid.forge.FluidStackHooksImpl.getStillTexture(FluidStackHooksImpl.java:81) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at dev.architectury.hooks.fluid.FluidStackHooks.getStillTexture(FluidStackHooks.java) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:epic_rei_lmao:A}
-- Screen details --
Details:
	Screen name: me.shedaniel.rei.impl.client.gui.error.ErrorsScreen
Stacktrace:
	at me.shedaniel.rei.impl.client.util.CrashReportUtils.screen(CrashReportUtils.java:50) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.util.CrashReportUtils.essential(CrashReportUtils.java:44) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.widget.BatchedEntryRendererManager.add(BatchedEntryRendererManager.java:90) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.widget.BatchedEntryRendererManager.addAll(BatchedEntryRendererManager.java:61) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.widget.BatchedEntryRendererManager.<init>(BatchedEntryRendererManager.java:55) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.widget.entrylist.EntryListWidget.m_6305_(EntryListWidget.java:314) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.ScreenOverlayImpl.renderWidgets(ScreenOverlayImpl.java:741) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.impl.client.gui.ScreenOverlayImpl.m_6305_(ScreenOverlayImpl.java:640) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at me.shedaniel.rei.RoughlyEnoughItemsCoreClient.lambda$registerEvents$20(RoughlyEnoughItemsCoreClient.java:393) ~[RoughlyEnoughItems-9.0.480.jar%2389!/:?] {re:classloading,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at dev.architectury.event.forge.EventHandlerImplClient.eventDrawScreenEvent(EventHandlerImplClient.java:158) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at dev.architectury.event.forge.__EventHandlerImplClient_eventDrawScreenEvent_Post.invoke(.dynamic) ~[architectury-5.6.24.jar%2375!/:?] {re:classloading,pl:eventbus:B}
	at net.minecraftforge.eventbus.ASMEventHandler.invoke(ASMEventHandler.java:93) ~[eventbus-6.0.0.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:302) ~[eventbus-6.0.0.jar%232!/:?] {}
	at net.minecraftforge.eventbus.EventBus.post(EventBus.java:283) ~[eventbus-6.0.0.jar%232!/:?] {}
	at net.minecraftforge.client.ForgeHooksClient.drawScreenInternal(ForgeHooksClient.java:383) ~[forge-1.19-41.0.33-universal.jar%2399!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:375) ~[forge-1.19-41.0.33-universal.jar%2399!/:?] {re:classloading,pl:epic_rei_lmao:A}
	at net.minecraft.client.renderer.GameRenderer.m_109093_(GameRenderer.java:896) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:classloading,pl:accesstransformer:B,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1071) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:661) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:architectury.mixins.json:MixinMinecraft,pl:mixin:A,pl:epic_rei_lmao:A,pl:runtimedistcleaner:A}
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.19-20220607.102129-srg.jar%2394!/:?] {re:classloading,pl:runtimedistcleaner:A}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?] {}
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?] {}
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?] {}
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?] {}
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:27) ~[fmlloader-1.19-41.0.33.jar%2316!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:30) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-10.0.1.jar%235!/:?] {}
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:155) [bootstraplauncher-1.1.0.jar:?] {}


-- Renderer details --
Details:
	Renderer name: me.shedaniel.rei.impl.common.entry.TypedEntryStack
	Z level: 0
	Entry type: minecraft:fluid
	Is empty: false
	Entry identifier: minecraft:lava
```

**Comment by MaxNeedsSnacks** [comment]
See #924, this is due to a breaking Forge change

**Comment by shedaniel** [comment]
See resolved issue in #924
